tests/puppeteer/campaigns/sanity/checkoutFO/01_orderProduct.js

```diff
@@ -0,0 +1,86 @@
+// Using chai
+const {expect} = require('chai');
+
+// Importing pages
+const HomePage = require('../../../pages/FO/home');
+const CartPage = require('../../../pages/FO/cart');
+const LoginPage = require('../../../pages/FO/login');
+const CheckoutPage = require('../../../pages/FO/checkout');
+const OrderConfirmationPage = require('../../../pages/FO/orderConfirmation');
+let customer = require('../../data/FO/customer');
+const CartData = require('../../data/FO/cart');
+
+let page;
+let homePage;
+let cartPage;
+let loginPage;
+let checkoutPage;
+let orderConfirmationPage;
+
+// creating pages objects in a function
+const init = async () => {
+  page = await global.browser.newPage();
+  await page.setExtraHTTPHeaders({
+    'Accept-Language': 'en-GB'
+  });
+  homePage = await (new HomePage(page));
+  cartPage = await (new CartPage(page));
+  loginPage = await (new LoginPage(page));
+  checkoutPage = await (new CheckoutPage(page));
+  orderConfirmationPage = await (new OrderConfirmationPage(page));
+};
+
+/*
+  Order a product and check order confirmation
+ */
+global.scenario('Order a product and check order confirmation', () => {
+  test('should open the shop page', async () => {
+    await homePage.goTo(global.URL_FO);
+    await homePage.checkHomePage();
+  });
+  test('should go to login page', async () => {
+    await homePage.goToLoginPage();
+    const pageTitle = await loginPage.getPageTitle();
+    await expect(pageTitle).to.equal(loginPage.pageTitle);
+  });
+  test('should sign In in FO With default account', async () => {
+    await loginPage.customerLogin(customer.defaultAccount);
+    const connected = await homePage.isCustomerConnected();
+    await expect(connected, 'Customer is not connected in FO').to.be.true;
```

Comment:
Why the sentence "is not connected in FO" should be true ? That means you're not connected, right ?